src/v5_hal/firmware/include/nodes/subsystems/drivetrain_nodes/HolonomicDriveNode.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "nodes/NodeManager.h"
+#include "api.h"
+#include "nodes/actuator_nodes/MotorNode.h"
+#include "nodes/sensor_nodes/ControllerNode.h"
+
+class HolonomicDriveNode : public Node {
+private:
+    MotorNode* m_left_front_motor;
+    MotorNode* m_left_rear_motor;
+    MotorNode* m_right_front_motor;
+    MotorNode* m_right_rear_motor;
+
+    std::string m_handle_name;
+
+    void m_setLeftFrontVoltage(int voltage);
+
+    void m_setLeftRearVoltage(int voltage);
+
+    void m_setRightFrontVoltage(int voltage);
+
+    void m_setRightRearVoltage(int voltage);
+
+    void m_setLeftFrontVelocity(int voltage);
+
+    void m_setLeftRearVelocity(int voltage);
+
+    void m_setRightFrontVelocity(int voltage);
+
+    void m_setRightRearVelocity(int voltage);
+
+public: 
+    HolonomicDriveNode(NodeManager* node_manager, std::string handle_name, MotorNode* left_front_motor, 
```

Comment:
For now we should probably pass in the controller node as well and have teleopPeriodic control the drivetrain (similar to TankDriveNode)